File: Source/Api/Service/ServiceListItem.cs
using System;
using System.Collections.Generic;
using ServiceRegister.Common;

namespace ServiceRegister.Api.Service
{
    public class ServiceListItem
    {
        public Guid Id { get; set; }
        public IEnumerable<LocalizedText> Names { get; set; }
        public string ServiceClasses { get; set; }
        public string OntologyTerms { get; set; }
    }
}

File: Source/AngularApplication/App/Models/ServiceListItem.ts
"use strict";

module ServiceRegister
{
    export class ServiceListItem
    {
        public name: string;

        constructor(public id: string, names: Array<LocalizedText>, public serviceClasses: string, public ontologyTerms: string)
        {
            if (names != null)
            {
                this.name = names[0].localizedValue;
            }
        }
    }
}  